btcwallet/btcwallet.go

```diff
@@ -0,0 +1,52 @@
+package btcwallet
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+)
```

Comment:
We can use `errors` package from standard lib instead

Reply:
Okay, once again I trusted the IDE